flare/gp_from_aimd.py

```diff
@@ -284,6 +299,11 @@ def train_gp(self):
                                self.gp.likelihood, self.gp.likelihood_gradient)
         self.train_count += 1
 
+        if self.checkpoint_interval \
```

Comment:
cool. This is nice. Maybe we should also add a variable "last_step_written" to record when the gp was dumped. So it won't dump twice at the end of the run.